wasp-ai/src/client/pages/ResultPage.jsx

```diff
@@ -393,6 +406,87 @@ function getCardinalNumber(number) {
+
+  useEffect(() => {
+    if (!!tokenNumberStr) {
+      const num = tokenNumberStr.slice(1, -1) * 1000
+      setTokens(num)
+    }
+  }, [tokenNumberStr])
+
+  useEffect(() => {
+    if (isStarRepoOpen) {
+      jsConfetti.addConfetti({
+        emojis: ['🐝'],
+        emojiSize: 120,
+      })
+    }
+  }, [isStarRepoOpen])
+  return (
+    <MyDialog isOpen={isStarRepoOpen} onClose={() => setIsStarRepoOpen(false)} title={<span>Your App is Ready! 🎉</span>}>
+      <div className="mt-6 space-y-5">
+        <p className="text-base leading-relaxed text-gray-500">
+          We've made this tool completely <span className="font-semibold">free</span> and cover all the costs 😇
+        </p>
+        <table className="bg-slate-50 rounded-lg divide-y divide-gray-100 w-full text-base leading-relaxed text-gray-500 text-sm">
+          {/* <caption>Fun Stats</caption> */}
+          <tr>
+            <td className="p-2 text-gray-600"> Number of tokens your app used: </td>
+            <td className="p-2 text-gray-600">
+              {" "}
+              <FormattedText>{tokens.toLocaleString(2) ?? "~22k"}</FormattedText>{" "}
+            </td>
+          </tr>
+          <tr>
+            <td className="p-2 text-gray-600"> Cost to generate your app: </td>
+            <td className="p-2 text-gray-600">
+              {" "}
+              <FormattedText>{tokens ? `$${((tokens / 1000) * 0.004).toFixed(2)}` : "~ $0.50"}</FormattedText>{" "}
+            </td>
+          </tr>
+          {numTotalProjects && (
+            <tr className="p-2 text-gray-600">
+              <td className="p-2 text-gray-600"> Total number of apps generated with Mage: </td>
+              <td className="p-2 text-gray-600">
+                {" "}
+                <FormattedText>{numTotalProjects.toLocaleString()}</FormattedText>{" "}
+              </td>
+            </tr>
+          )}
+        </table>
+        <p className="text-base leading-relaxed text-gray-500">
+          But you can still show your support by starring us on GitHub:
+        </p>
+        <a
+          href="https://github.com/wasp-lang/wasp"
+          target="_blank"
+          className="flex items-center justify-center underline text-pink-600 "
+        >
+          <div className="py-4 px-2 flex items-center justify-center bg-pink-50 text-pink-800 rounded-lg font-semibold tracking-wide w-full">
+            <PiStarDuotone size="1.35rem" className="mr-3" /> Star Wasp on GitHub{" "}
+            <PiGithubLogoDuotone size="1.35rem" className="ml-3" />
+          </div>
+        </a>
+        <p className="text-base leading-relaxed text-gray-500">
+          This helps spread the word, so we can keep making Mage better.
+        </p>
+        <p className="text-base leading-relaxed text-gray-500">We'd very much appreciate it! 🧙</p>
+      </div>
+    </MyDialog>
+  )
+}
+
+function FormattedText({ children }) {
```

Comment:
I would probably put this one inside `StarOurRepoModal` since it is used only there and is very specific just for it.